source/includes/gcp/_k8_releases.md

```diff
@@ -319,7 +319,31 @@ Attributes | &nbsp;
 `version`<br/>*string* | The revision of the release
 `namespace`<br/>*string* | The namespace to which the release is installed
 
+<!-- ROLLBACK RELEASE -->
+### Rollback release to previous revision
 
+```shell
+curl -X POST \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/k8s/anenvironment/releases/pspensieri/aerospike-1579797954&operation=rollback"
```

Comment:
it is missing the clusterid parameter. Please check the other one from gcp to conform to it

Reply:
Added the `clusterid` in the urls and in the parameter tables, see https://github.com/cloudops/cloudmc-api-docs/pull/86/commits/7cee492164ec7e3e4994afdc992e4ad5a4677f19